testing.conf

```diff
@@ -0,0 +1,37 @@
+http {
+  interface="localhost"
+  port=8080
+}
+
+vault {
+  server="https://api-ci.vault.broadinstitute.org/api"
+  analysesIngestPath="/analyses"
+  analysesDescribePath="/analyses/%s"
+  analysesUpdatePath="/analyses/%s/outputs"
+  analysesRedirectPath="/analyses/%s/%s"
+  lookupPath="/query/%s/%s/%s"
+  ubamsIngestPath="/ubams"
+  ubamsDescribePath="/ubams/%s"
+  ubamsRedirectPath="/ubams/%s/%s"
+}
+
+dm {
+  server="http://localhost:8989"
+  ubamsPath="/ubams"
+  ubamsResolvePath="/ubams/%s"
+  analysesPath="/analyses"
+  analysesResolvePath="/analyses/%s"
+  analysesUpdatePath="/analyses/%s/outputs"
+  queryLookupPath="/query/%s/%s/%s"
+}
+
+boss {
+  server="http://localhost:8988"
+  objectsPath="/objects/"
+  objectsUrl="https://boss-dev.broadinstitute.org/objects/"
+  objectResolvePath="/resolve"
+  defaultStoragePlatform = "cloudStore"
+  defaultValidityPeriodSeconds = 1000
+  bossUser=""
+  bossUserPassword=""
+}
```

Comment:
Let's move this file to src/test/resources. Our application.conf file should be moved to src/main/resources, but not for this PR.